resources/views/multiple-list-dropdown.blade.php

```diff
@@ -0,0 +1,30 @@
+@props([
+    'values',
+    'description' => null,
+])
+
+<div x-data="{ showDropdown: false }" x-cloak>
+    <div
+        class="multiple-list-link"
+        @click="showDropdown = true"
+        @click.away="showDropdown = false"
+    >
+        @lang('ui::general.multiple')
+    </div>
+
+    <div class="multiple-list-dropdown" x-bind:class="{ hidden: ! showDropdown }">
```

Comment:
isn't this just a regular dropdown but with custom content? 🤔 issue with creating a `multiple-list` component is that you can only use this for 2 columns, if you need 3 or 4 you end up having to create new components if you cannot simply pass a custom content property